### PR TITLE
Add private room for SagarV and Smoke Detector on meta chat

### PR DESCRIPTION
**Pull request by sagarvd01**
added a new private room only for reports. no other discussion.

### CODE REVIEW

**Comment by tripleee** [comment]
Could you document here the rationale for adding another room on Meta? Smoke Detector is already in the [Tavern](https://chat.meta.stackexchange.com/rooms/89/tavern-on-the-meta) (and posts are periodically archived to [Chimney](https://chat.meta.stackexchange.com/rooms/1037/chimney)). If this is just for your personal needs, you can set up Smokey to ping you in chat with `!!/notify`, or you can install the spam alert userscript.

**Comment by Glorfindel83** [comment]
@tripleee to be fair, !!/notify is rather broken.

**Comment by tripleee** [comment]
For the record, the Travis failure is just the run-of-the-mill `flake8` whitespace flak.

**Comment by AWegnerGitHub** [comment]
I'd also like to know the rational behind this.

File: ws.py [hunk 1]
```diff
@@ -197,6 +197,10 @@ def check_socket_connections():
         "room": GlobalVars.wrap.get_room("388"),
         "unwantedReasons": []
     },
+    {   "sites": ["meta.stackexchange.com"],
```
Comment (tripleee) [inline ws.py]: `flake8` dislikes this, but the fix should be trivial -- just add a newline after the opening brace and reindent.

File: ws.py [hunk 2]
```diff
@@ -197,6 +197,10 @@ def check_socket_connections():
...
+        "room": GlobalVars.wrap.get_room("1121"),
```
Comment (tripleee) [inline ws.py]: I *think* you will need to use `wrapm` rather than `wrap` to get the correct room.

**Comment by AWegnerGitHub** [comment]
Rejected due to lack of explanation both here and in CHQ

**Pull request closed without merging**